buildSrc/src/main/java/Configurations.kt

```diff
@@ -0,0 +1,32 @@
+object Flavors {
```

Comment:
@luongvo, I think we have a duplicate works with @kaungkhantsoe's works in this [PR](https://github.com/nimblehq/kmm-templates/pull/25)

Reply:
@phongvhd93 Yes, we can rebase for the following PRs. I think this PR should go first 🤷‍♂️ cc @kaungkhantsoe